bin/needs-a-genny

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+genny_path = ->(slug) {
+  ['exercises', slug, '.meta', 'generator'].join('/')
+}
+
+genny_path_exists = ->(slug) { !Dir.exist?(genny_path[slug]) }
```

Comment:
This method name is misleading, since you seem to be checking if the path does not exist.

Reply:
I looked at that 3 times and missed this, definitely correct.